.github/workflows/macos.yml

```diff
@@ -21,26 +21,27 @@ jobs:
         build:
         - name: normal
           install: nghttp2
+          configure: --without-ssl
           macosx-version-min: 10.9
         - name: debug
           install: nghttp2
-          configure: --enable-debug --enable-mqtt
+          configure: --enable-debug --enable-mqtt --without-ssl
```

Comment:
MQTT is enabled by default, right? So that part could be removed here.

Reply:
Thanks, addressed all three of them now!